bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard tiles
+
+OpenHAB dashboard is landing page for the user where all openHAB UI's can be found. Dashboard support also links to external services. Links can be added to dashboard by ```conf/services/dashboard.cfg``` configuration file.
+
+## Link Configuration
+
+| Parameter name  | Type    | Description                                                                             |
+|-----------------|---------|-----------------------------------------------------------------------------------------|
+| link-nameX      | String  | Name which is shown in the openHAB dashboard.                                           |
+| link-urlX       | String  | URL to external service.                                                                |
+| link-overlayX   | String  | Image overlay icon. Supported values are empty (no icon), "html5", "android" or "apple" |
+| link-imageurlX  | String  | URL to image.                                                                           |
+
+Where X is link unique number (see examples). All configuration parameters need to start with ```org.openhab.ui.dashboard:``` prefix.
```

Comment:
That was the missing piece. I was wondering why some of the configuration files works without prefix.